tests/phpunit/tests/oembed/getResponseData.php

```diff
@@ -2,6 +2,7 @@
 
 /**
  * @group oembed
+ * @covers ::get_oembed_response_data
```

Comment:
Minor tidy up of tests from previous patch:
* Add `@covers`
* renamed a couple tests to mirror function name
* fixed a couple of typos
* removed manual calls to unregister the custom statuses as the test suite does that already